server/V2/controllers/EntryController.js

```diff
@@ -21,4 +27,30 @@ export default class Entry {
       return Response.errorResponse(res, 500, `${err.message}`);
     }
   }
+
+  /**
+ *@description handles retrieve all entries
+ *
+ * @param {object} req
+ * @param {object} res
+ */
```

Comment:
You can add `@returns [array] All entries` after line 35 for example:
<img width="302" alt="Screen Shot 2019-11-05 at 12 37 46" src="https://user-images.githubusercontent.com/9946845/68200770-325d5e80-ffc9-11e9-98d6-00b311132460.png">

Reply:
You can add what is returned in your `docstrings`